hexapod_dart/src/hexapod_dart_simu.cpp

```diff
@@ -334,3 +291,86 @@ void HexapodDARTSimu::_add_floor()
 
     _world->addSkeleton(floor);
 }
+
+double HexapodDARTSimu::_min_dist_angle(double a1, double a2)
```

Comment:
I don't see how this algorithm works if you have negative angles lower than -180°.

For instance, if we have -190°:

- add 180° -> -10°
- remove any multiplier of 360° -> -10°
- remove 180° -> -190°

Did I misunderstand ?